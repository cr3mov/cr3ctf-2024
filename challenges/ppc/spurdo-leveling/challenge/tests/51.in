135586271857830404 943607738876528308
4 3
3 701
