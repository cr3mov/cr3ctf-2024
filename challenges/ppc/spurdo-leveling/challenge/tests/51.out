1151027730795867 2 1151027730795866
